docs/signatory-architecture.md

```diff
@@ -0,0 +1,295 @@
+---
+id: architecture
+title: Signatory Architecture
+sidebar_label: Signatory Architecture
+---
+
+# Signatory C4 Architecture Model 
+
+Signatory is the perfect solution for secure and reliable remote signing in Tezos Blockchain cases. With easy-to-use configuration options, users can set up vaults and policies to ensure that only authorized operations are signed. Signatory also supports a variety of hardware-based and cloud-based HSMs, such as [AWS KMS](https://aws.amazon.com/kms/) and [YubiHSM](https://www.yubico.com/?utm_source=google&utm_medium=pd:search&utm_campaign=US_B2C_LeadGen_Google_SEM_Brand&utm_content=&gclid=CjwKCAiAmJGgBhAZEiwA1JZolnAOjpSi_GVY8qz7NyLHASwkXHBu2t1aPNWl2WWHl4Nb2S19OsAWMBoCGucQAvD_BwE), to protect cryptographic keys.
+
+Using Signatory, users can securely store their secret keys and control which operations can be signed, reducing the risk of losing or having their keys stolen. Signatory also allows users to sign transactions on hardware not connected to the internet, providing an additional layer of security.
+
+[C4 architecture diagrams](https://c4model.com/) are visual modelling languages used to communicate software system architectures. They are designed to be simple, easy to understand and provide a high-level view of a system's components and how they interact. C4 diagrams are typically organized into a hierarchy of increasing detail, with each view building on the previous one.
+
+[Domain-driven design](https://martinfowler.com/tags/domain%20driven%20design.html) ([DDD](https://learn.microsoft.com/en-us/archive/msdn-magazine/2009/february/best-practice-an-introduction-to-domain-driven-design)) is a software development approach emphasizing the importance of understanding the business domain to create effective software solutions. C4 diagrams can help developers understand the domain of the system they are building and design software that aligns with the domain concepts and principles. By using C4 diagrams to visualize the system's architecture, developers can better understand how the system fits into the larger business context and how the different components of the system interact with one another.
+
+The C4 system architectural model is a set of diagrams used to represent the architecture of a software system. There are four levels in the C4 model, each providing an increasing level of detail about the software system.
```

Comment:
this paragraph is kind of a repeat of 2 paragraphs prior. certainly the "increasing level of detail" is repeated.

Reply:
removed the repetitive clause ", each providing an increasing level of detail about the software system."